fic_modules/hg.py

```diff
@@ -302,10 +306,12 @@ def extract_json_from_hg(json_files, path_to_files, days_to_generate):
                                         review,
                                         commit_date)
             except AttributeError:
-                logger.info("Attribute Error!! \n "
-                            "Probable issue is an malfunctioned json file.. "
-                            "Please check the following file:{}".format(file))
+                LOGGER.exception("Attribute Error!! \n Probable issue is an "
+                                 "malfunctioned json file.. Please check the "
+                                 "following file:{}".format(file))
             except KeyError:
-                print("File {}is empty. \n",
-                      "Please check:{}",
-                      " for more details.\n".format(file, repository_url))
+                LOGGER.exception("File {} is empty. Please check: {} for more "
+                                 "details.".format(file, repository_url))
+                # print("File {}is empty. \n",
```

Comment:
If we don't require this anymore, it should be removed.

Reply:
Fixed.